src/components/input/_macro-options.md

```diff
@@ -28,3 +29,17 @@
 | text  | string | true     | The text for the prefix/suffix                                                                   |
 | title | string | true     | The title of the prefix/suffix. For example where `text` is "cm", `title` would be "centimeters" |
 | id    | string | false    | Id for the prefix/suffix                                                                         |
+
+## Typeahead
+
+| Name          | Type                                 | Required | Description                                                              |
+| ------------- | ------------------------------------ | -------- | ------------------------------------------------------------------------ |
+| id            | string                               | true     | ID for the input                                                         |
+| classes       | string                               | false    | CSS classes to apply to the wrapping element                             |
+| content       | TypeaheadContent                     | true     | Aria and results messaging content                                       |
```

Comment:
This object is missing documentation.

Reply:
All aria messaging needs to be checked with Content designers.  I know 'type' should be replaced with 'enter'